front_end/content.py

```diff
@@ -363,7 +387,7 @@ def unregister_user_from_course(self, course_id, user_id):
 
     def check_user_registered(self, course_id, user_id):
         sql = '''SELECT *
```

Comment:
It's a minor thing, but it's a little more efficient if you do SELECT 1 instead of SELECT *. Then it won't have to pull values from all columns in the table. It can just use 1 to indicate when there are rows returned from the query.